### PR TITLE
Add project brief readme

### PR DESCRIPTION
**Pull request by PiotrBerebecki**
Include info about the task and goals for the week-8 project

relates #464

### CODE REVIEW

**Comment by eliascodes** [comment]
My initial reaction is that we could judge this based on how the students have reacted to the other open-ended projects in the course. That will be different cohort to cohort, so it may well be useful to have one or more example ideas for an app that can be used if a cohort prefers more direction?

File: coursebook/week-8/project.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+### Goals
+
+The primary aim of this project is to get comfortable with setting up an Express server and to discover the awesomeness of Handlebars! Spend time considering how to format your helper functions, layouts, partials and views.
+
+### What's also important:
+
+- Include tests and set up code coverage.
+- Use Heroku or a similar service to host the app and the database.
+- Try to include ES6 syntax.
```
Comment (des-des) [inline coursebook/week-8/project.md]: (on the server)

Comment (PiotrBerebecki) [inline coursebook/week-8/project.md]: I've just clarified the info about [ES6 syntax on the server](https://github.com/foundersandcoders/master-reference/pull/492/files#diff-afe4733abefa3993dbd007e36509a97bR21)

**Comment by PiotrBerebecki** [comment]
I've just added [a sentence about about apps built by previous cohorts](https://github.com/foundersandcoders/master-reference/pull/492/commits/c221a1ebaa627f8e374fa5ce064c440d5aa4a86f#diff-afe4733abefa3993dbd007e36509a97bR15)

**Pull request merged**